dtreeviz/trees.py

```diff
@@ -585,10 +585,10 @@ def get_leaves():
 
         # TODO do we need show_edge_labels ?
         show_edge_labels = False
-        all_llabel = '&lt;' if show_edge_labels else ''
-        all_rlabel = '&ge;' if show_edge_labels else ''
-        root_llabel = self.shadow_tree.get_root_edge_labels()[0] if show_root_edge_labels else ''
-        root_rlabel = self.shadow_tree.get_root_edge_labels()[1] if show_root_edge_labels else ''
+        all_llabel = '  &lt;' if show_edge_labels else ''
```

Comment:
Do we have a before and after picture the demonstrates why the extra space is needed?

Reply:
I don't have a before an after at the ready, but when the tree is large / wide and the edge arrows are spread apart, the inequality symbols can overlap the arrows. This gives some extra room to help avoid that.